download_comments.py

```diff
@@ -72,12 +72,14 @@ def get_more_comments(start_id, users):
 
     return local_max_id, comments
 
-
-def download_comments():
+# Adding cookies and headers to definition since we won't get from auth.py
+def download_comments(Cookies, Headers):
+    cookies = Cookies
+    headers = Headers
```

Comment:
```suggestion
def download_comments(cookies, headers):
```
Would this also work the same way as in other functions?

Reply:
That should not have made it into the version I pushed. I made changes after originally writing that part and was sure I did a git add after. I'll check that out in my next commit.

Reply:
Cleaned up.